src/controllers/commentsController.js

```diff
@@ -0,0 +1,51 @@
+import sequelize from 'sequelize';
+import models from '../database/models';
+import responseUtil from '../utils/responseUtil';
+import strings from '../utils/stringsUtil';
+
+export default class CommentsController {
+  static editComment(req, res) {
+    const { Op } = sequelize;
+    const { id } = req.params;
+    const ownerId = req.user.payload.id;
+    const { comment } = req.body;
+
+    models.comments.findOne({
+      where: {
+        [Op.and]: [{ id },
+          { deleted: false }]
+      },
+    }).then(comments => {
+
+      if (!comments) {
+        return responseUtil(res, 404, strings.comments.error.COMMENT_NOT_FOUND);
+      }
+      if (comments.userId !== ownerId) {
+        return responseUtil(res, 403, strings.comments.error.NOT_OWNER);
+      }
+      models.comments.update({ comment, }, { where: { id, }, });
+      return responseUtil(res, 200, strings.comments.success.COMMENT_UPDATED);
+    });
+  }
```

Comment:
your methods to `edit` and to `delete` are very much similar.
Kindly refactor it.
**Suggestion:** 
- Create a helper to find a comment, by passing the required parameters in the helper
- A helper to check if **not found** and **not owner**
- Then one to update by passing the required parameters in the helper.